src/univariate/continuous/laplace.jl

```diff
@@ -70,6 +70,11 @@ skewness(d::Laplace{T}) where {T<:Real} = zero(T)
 kurtosis(d::Laplace{T}) where {T<:Real} = 3one(T)
 
 entropy(d::Laplace) = log(2d.θ) + 1
+        
+function kldivergence(p::Laplace, q::Laplace)
+    r = abs(p.μ - q.μ)
+    return (p.θ * exp(-r / p.θ) + r) / q.θ + log(q.θ / p.θ) - 1
```

Comment:
Not sure if we gain much (the compiler might take care of it?), but I just noticed that we can avoid repeated and explicit field accesses here:

```suggestion
    pμ, pθ = params(p)
    qμ, qθ = params(q)
    r = abs(pμ - qμ)
    return (pθ * exp(-r / pθ) + r) / qθ + log(qθ / pθ) - 1
```

Reply:
The LLVM is exactly the same between two version. I would propose to keep the more concise version.

Reply:
I'm not too surprised that the compiler generates the same code in both cases. The main advantage here would be that there are no explicit field accesses, so it's easier to rename the parameters and not necessary to know the internals of the struct. 

But yeah, this is not a major blocking issue.

Reply:
I hadn't thought of that. Makes sense for maintainability.